README.md

```diff
@@ -14,11 +14,10 @@ Tekton Dashboard is a general purpose, web-based UI for Tekton Pipelines. It all
 
 ### Which version should I use?
 
-It is strongly recommended to use the *v0.6.1.2* release for Tekton Pipelines v0.11.x (can display components from Tekton Triggers 0.4.x). This is an important security release.
+It is strongly recommended to use the *v0.6.1.3* release for Tekton Pipelines v0.11.x and v0.12.x (can display components from Tekton Triggers 0.4.x). This is an important security release.
```

Comment:
(can display components from Tekton Triggers 0.4.x) doesn't read well. Can the grammar be polished up here please?